docs/coder_tls_acme.md

```diff
@@ -18,9 +18,9 @@ tls acme --email me@example.com --domains a.example.com --domains b.example.com
 
 ```
       --agree-tos                    Agree to ACME Terms of Service
-  -c, --credentials stringToString   DNS provider credentials (default [])
+      --credentials stringToString   DNS provider credentials (default [])
       --domains stringArray          Domains to request certificates for
```

Comment:
How does pflag accept arrays? Are they specified like "[coder.com,abc.com]" or some other way?

Reply:
You have to specify them one by one (`--domains a --domains b`)
It took me a while to figure that out, so I provide an example for the `tls acme` command


`coder tls acme -h` shows

```tls acme --email me@example.com --domains a.example.com --domains b.example.com --provider route53 --credentials AWS_ACCESS_KEY_ID=your-key-id --credentials AWS_SECRET_ACCESS_KEY=your-secret-key --credentials AWS_REGION=your-region```

Reply:
Ahh I see. Do you mind adding a similar example for domains?

Reply:
Ah nevermind, your example is for the full subcommand, alright then, carry on!